test/test_rails_view_render_literal.rb

```diff
@@ -54,41 +54,33 @@ def test_render_component_no_offense
     assert_equal 0, cop.offenses.count
   end
 
-  def test_render_component_class_name_no_offense
-    erb_investigate cop, <<-ERB, "app/views/foo/index.html.erb"
-      <%= render Module::MyClass, title: "foo", bar: "baz" %>
-    ERB
-
-    assert_equal 0, cop.offenses.count
-  end
-
-  def test_render_component_class_no_offense
+  def test_render_component_instance_no_offense
     erb_investigate cop, <<-ERB, "app/views/foo/index.html.erb"
-      <%= render MyClass, title: "foo", bar: "baz" %>
+      <%= render MyClass.new(title: "foo", bar: "baz") %>
     ERB
 
     assert_equal 0, cop.offenses.count
   end
 
-  def test_render_component_instance_no_offense
+  def test_render_component_instance_block_no_offense
     erb_investigate cop, <<-ERB, "app/views/foo/index.html.erb"
-      <%= render MyClass.new(title: "foo", bar: "baz") %>
+      <%= render Module::MyClass.new(title: "foo", bar: "baz") do %>Content<% end %>
     ERB
 
     assert_equal 0, cop.offenses.count
   end
 
-  def test_render_component_instance_block_no_offense
+  def test_render_component_collection_no_offense
     erb_investigate cop, <<-ERB, "app/views/foo/index.html.erb"
-      <%= render Module::MyClass.new(title: "foo", bar: "baz") do %>Content<% end %>
+      <%= render MyClass.collection(title: "foo", bar: "baz") %>
     ERB
 
     assert_equal 0, cop.offenses.count
   end
 
-  def test_render_component_class_block_no_offense
+  def test_render_component_module_collection_no_offense
     erb_investigate cop, <<-ERB, "app/views/foo/index.html.erb"
-      <%= render Module::MyClass, title: "foo", bar: "baz" do %>Content<% end %>
+      <%= render Foo::MyClass.collection(title: "foo", bar: "baz") %>
```

Comment:
What's the difference between `collection` and `with_collection`?

Reply:
🤦 You're right, I used the wrong method name!